sbt_common/internal_model/src/main/resources/location-types.csv

```diff
@@ -1,5 +1,6 @@
 ThumbnailImage,thumbnail-image,Thumbnail Image
-IIIFImage,iiif-image,IIIF image
+IIIFImage,iiif-image,IIIF Image API
+IIIFPresentation,iiif-presentation,IIIF Presentation API
```

Comment:
This file also lives in the ontologies folder, under Reference Data. It needs updating there as well.

(Yes, the duplication is daft, but not something to fix today.)